index.js

```diff
@@ -1,7 +1,37 @@
-const probe = require('probe-image-size');
+const debug = require('debug')('iiif-processor:main');
```

Comment:
does this and the debug calls need to be removed or is it smart about loading it only for dev environments?

Reply:
looked this up. learned a coold new thing. I withdraw the question